packages/relay-compiler/codegen/RelayFileWriter.js

```diff
@@ -23,7 +23,7 @@ const graphql = require('graphql');
 const invariant = require('invariant');
 const nullthrows = require('nullthrows');
 const path = require('path');
-const writeRelayGeneratedFile = require('./writeRelayGeneratedFile');
+const {writeRelayGeneratedFile} = require('./writeRelayGeneratedFile');
```

Comment:
Yeah, this, unfortunately, won't work with our internal FB compiler setup. 

So we should probably keep default export from this `writeRelayGeneratedFile`.